client/src/BusinessCard/BusinessCard.js

```diff
@@ -55,6 +55,7 @@ class BusinessCard extends Component {
     let data = this.retrieveData();
     let urlParams = Object.entries(data).map(e => e.join('=')).join('&');
     this.setState({ loading: true });
+    this.setState({ mainBusiness: null });

```

Comment:
the setState should contain these two state instead of calling the setState twice

Reply:
Resolved

Reply:
Thank you